platforms/gke/base/use-cases/federated-learning/terraform/example_cross_device/templates/cross_device_authorization_policies.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2025 Google LLC
```

Comment:
A lot of these descriptors are duplicated if you consider the `example_nvidia_flare_tff` terraservice. Extract the duplicated ones from that terraservice an place them in a common module, and add the new terraservice to the list of terraservices to provision for both cross device and the nvflare example.